api/v1alpha3/azurecluster_conversion.go

```diff
@@ -75,6 +75,15 @@ func (src *AzureCluster) ConvertTo(dstRaw conversion.Hub) error {
 
 	dst.Spec.NetworkSpec.APIServerLB.FrontendIPsCount = restored.Spec.NetworkSpec.APIServerLB.FrontendIPsCount
 	dst.Spec.NetworkSpec.APIServerLB.IdleTimeoutInMinutes = restored.Spec.NetworkSpec.APIServerLB.IdleTimeoutInMinutes
+
+	for _, restoredFrontendIP := range restored.Spec.NetworkSpec.APIServerLB.FrontendIPs {
+		for i, dstFrontendIP := range dst.Spec.NetworkSpec.APIServerLB.FrontendIPs {
+			if restoredFrontendIP.Name == dstFrontendIP.Name && restoredFrontendIP.PublicIP != nil {
```

Comment:
Is there a case where the front end IPs wouldn't be in the same order after restoring them? If we can't rely on them being in the same order, could we construct a map from the `dstFrontendIP.Name` to its index so we don't have to search using a nested for loop?

Reply:
we're doing a nested for loop and matching on `Name` so order should not matter